students/18th/18기 양지은/login.css

```
@@ -0,0 +1,147 @@
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100vh;
+}
+.container {
+  width: 350px;
+  height: 400px;
+  border: 1px solid lightgray;
+}
+.westagram {
+  font-family: "Lobster", cursive;
+  text-align: center;
+  font-size: 45px;
+  margin: 40px;
+}
+.bigbox {
+  display: flex;
+  flex-direction: column;
+}
+
+.inputs {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+input {
+  width: 250px;
+  height: 30px;
+  margin: 3px;
+  padding: 3px;
+  padding-left: 8px;
+  font-size: 11px;
+  border: 1px solid lightgray;
+  border-radius: 3px;
+}
+
+.btn-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.btn {
+  width: 260px;
+  height: 32px;
+  margin: 15px;
+  background-color: lightskyblue;
+  color: white;
+  font-weight: bold;
+  font-size: 14px;
+  border: 0;
+  border-radius: 3px;
+}
+
+.or {
+  display: flex;
+  flex-direction: low;
+  justify-content: center;
+  margin-top: 10px;
+}
+
+.line-l {
+  margin-top: 5px;
+  margin-bottom: 20px;
+  width: 95px;
+  border: 0.5px solid lightgray;
+}
+
+.centerchar {
+  margin-left: 20px;
+  margin-right: 20px;
+  font-size: 13px;
+  font-weight: bold;
+  color: lightslategray;
+}
+
+.line-r {
+  margin-top: 5px;
+  margin-bottom: 20px;
+  width: 95px;
+  border: 0.5px solid lightgray;
+}
+
+.facebook {
+  display: flex;
+  justify-content: center;
+  margin-top: 15px;
+  margin-bottom: 15px;
```

css순서 예쁘게 잘 작성해주셨네요! 
margin값은 한 줄로 작성해주시면 코드를 더 간결하게 줄일 수 있습니다.
```suggestion
  display: flex;
  justify-content: center;
  margin: 15px 0;
```

---

넵 보완하겠습니다!